pyRDDLGym_jax/core/planner.py

```diff
@@ -1969,7 +2018,11 @@ def optimize_generator(self, key: Optional[random.PRNGKey]=None,
             # stopping condition reached
             if stopping_rule is not None and stopping_rule.monitor(callback):
                 break                
```

Comment:
I know that it is out of context for this PR, but we could set the status here to something like `JaxPlannerStatus.STOPPING_RULE_REACHED` to finish the optimization with an accurate status on the last callback (today it returns `JaxPlannerStatus.NORMAL`).

Reply:
Yes, indeed! I have added a `STOPPING_RULE_CONVERGED` flag that should be displayed when the rule is activated. I also fixed the progress bar showing an outdated status code.